src/InputManager.cpp

```diff
@@ -0,0 +1,34 @@
+#include "InputManager.h"
+
+InputManager::InputManager(GLFWwindow *w, Camera* cam)
+:window(w)
+,camera(cam)
+{
+}
+
+void InputManager::CheckInput(){
+	KeyInput();
+	MouseInput();
+}
+
+void InputManager::MouseInput() {
+	double xpos, ypos;
+	if(glfwGetMouseButton(window, GLFW_MOUSE_BUTTON_LEFT) == GLFW_PRESS){
+		// Get mouse position
+		glfwGetCursorPos(window, &xpos, &ypos);
+		float deltaX = (xpos - cursorX) * 0.01;
+		float deltaY = (ypos - cursorY) * 0.01;
```

Comment:
Why 0.01? Please name it a constant so we know where it came from and can change it easily.

Also, in c++, 0.01 is a double which then has to unnecessarily cast to float before multiplication with another float.

Reply:
d